### DISCUSSION TITLE
User oscilators NTS1 style

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#276] [Ideas]
Could this be an option ? I don't know how oscilators are defined or programmed but I was wondering if it could be possible to define user oscilators, like a Chord oscilator on the nts1 which uses just 1 note of polyphony, but it contains all the harmonics of a chord

**Comment by bangcorrupt** [comment]
I don't know a lot about the oscillators yet, only changing their parameters.  We can make chord oscillators using single cycle waveform samples.  Hopefully in the future we will be able to build new analogue modeled oscillators from the sound engine.
